Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -0,0 +1,583 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Graph;
+
+namespace Gordon360.Services.RecIM
+{
+    public class SeriesService : ISeriesService
+    {
+        private readonly CCTContext _context;
+        private readonly IMatchService _matchService;
+
+        public SeriesService(CCTContext context, IMatchService matchService)
+        {
+            _context = context;
+            _matchService = matchService;
+        }
+        public IEnumerable<LookupViewModel> GetSeriesLookup(string type)
+        {
+            if (type == "status")
+            {
+                var res = _context.SeriesStatus
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
+            if (type == "series")
+            {
+                var res = _context.SeriesType
+                    .Select(s => new LookupViewModel
+                    {
+                        ID = s.ID,
+                        Description = s.Description
+                    })
+                    .AsEnumerable();
+                return res;
+            }
```

Comment:
More type codes that could all be in one 'code table'.

Reply:
refer to above